src/components/SettingsPage/sections/Currencies.js

```diff
@@ -29,30 +31,11 @@ import {
   SettingsSectionRow as Row,
 } from '../SettingsSection'
 
-//  .
-// /!\ Please note that this will likely not be like that in the future.
-//     I guess that all currencies should have those settings inside them
-//     instead of using same default for all.
-//
-const CURRENCY_DEFAULTS_SETTINGS: CurrencySettings = {
-  confirmationsToSpend: 10,
-  minConfirmationsToSpend: 10,
-  maxConfirmationsToSpend: 50,
-
-  confirmationsNb: 10,
-  minConfirmationsNb: 10,
-  maxConfirmationsNb: 50,
-
-  transactionFees: 10,
-
-  exchange: '',
-}
-
 type Props = {
   counterValueCurrency: Currency,
   currencies: CryptoCurrency[],
-  settings: Settings,
-  saveSettings: Function,
+  settings: SettingsState,
+  saveSettings: ($Shape<SettingsState>) => void,
```

Comment:
we have to update all similar `Function` types everywhere in the app, we have it all over the place